en/Search Algorithms/Floyd Cycle Detection Algorithm to Find Duplicate Number.md

```diff
@@ -0,0 +1,40 @@
+# Floyd Cycle Detection Algorithm to find duplicate number in an array
+
+#### Problem Statement
+
+Given an array of integers containing 'n + 1' integers, where each integer is in the range [1, n] inclusive. If there is only one duplicate number in the input array, this algorithm returns the duplicate number without modifying the original array, otherwise, it returns -1.
```

Comment:
```suggestion
Given an array of integers containing `n + 1` integers, where each integer is in the range [1, n] inclusive. If there is only one duplicate number in the input array, this algorithm returns the duplicate number without modifying the original array, otherwise, it returns -1.
```